sr/robot/camera.py

```diff
@@ -55,7 +58,7 @@ def _discover(cls) -> Dict[str, 'AprilCamera']:
         return {
             (serial := f"{camera_data.name} - {camera_data.index}"):
             cls(camera_data.index, camera_data=camera_data, serial_num=serial)
-            for camera_data in find_cameras(calibrations)
+            for camera_data in find_cameras(robot_calibrations)
```

Comment:
Suggestion: Should we also include the user code directory here so competitors can easily add their own calibrations?

Reply:
`calibrations` already includes this, I was inserting the sr-robot calibrations below the april_vision calibrations though so I've corrected that.
bfbb2e36e3636eb9bf679f5664b74761d216df92